vars/setupCommonEnvironmentVariables.groovy

```diff
@@ -138,7 +138,7 @@ def call() {
             env.HELPERNODE_REPO = "https://github.com/RedHatOfficial/ocp4-helpernode"
             env.HELPERNODE_TAG = "73312da6192298b3174d54e0a2ae129429561f77"
```

Comment:
We do not need to overwrite default helpernode tag as well.

Reply:
After removing tags both PowerVS and PowerVC jobs are working